policy/pa.go

```diff
@@ -412,8 +412,10 @@ func (pa *AuthorityImpl) checkHostLists(domain string) error {
 }
 
 // ChallengesFor makes a decision of what challenges, and combinations, are
-// acceptable for the given identifier.
-func (pa *AuthorityImpl) ChallengesFor(identifier core.AcmeIdentifier, regID int64) ([]core.Challenge, [][]int, error) {
+// acceptable for the given identifier. If the TLSSNIRevalidation feature flag
+// is set, create TLS-SNI challenges for revalidation requests even if TLS-SNI
```

Comment:
nit: TLS-SNI should be TLS-SNI-01 here. We only check if `core.ChallengeTypeTLSSNI01` is enabled/disabled, might as well be specific